.github/workflows/qe-tests.yml

```diff
@@ -0,0 +1,114 @@
+name: QE Tests
+
+on: [push, pull_request]
+
+jobs:
+    # "Build" job
+    build:
+      # This job tries to build PKI from src on a fresh docker container.
+      # The docker container is spawned by github itself and we merely just
+      # run the build commands. We then upload the artifact for consumption
+      # by the test jobs + for the public to consume. This job **does not**
+      # upload any build logs as they are visible in the log console itself.
+
+      name: Build PKI
```

Comment:
This builds PKI again, right? In addition to our existing builds? Is there a way to share builds across different jobs, and stick them in a single, common place?

Reply:
There is no such feature yet. Each workflow is unrelated to other workflows